packages/main/src/resolvers/conversation.ts

```diff
@@ -116,6 +116,26 @@ export const ConversationMutations: ConversationMutationsResolvers = {
     return threads
   },
 
+  async flagPresentable(_parent, { conversationId, id, isFlagged }) {
+    const thread = C.mustGetConversation(conversationId)
+    for (const message of thread.messages) {
+      if (String(message.id) === id) {
```

Comment:
This will probably work when using Poodle. But this will not properly unstar edit messages that were starred from Gmail. The id of the presentable happens to be the cache id of the message that provides the first revision of the presentable. But there is a cleaner way to do this: call `getPresentableElements(thread)`, find the presentable with the matching id, and get the list of messages making up that presentable from the presentable object itself. As I said before that involves adding a list of `revisions` to the `Presentable` type.

For purposes of interoperation with other email clients I think it would make most sense to add a flag the latest edit message instead of the first revision. When removing flags it is important to remove from all messages in the presentable revision history that have flags set.

I'd like to see tests to make sure that starring and unstarring works with presentables that have been edited.